### ISSUE TITLE
Damaged MB??

### ISSUE THREAD

**Issue by ClarkSteel** [hirschmann/nbfc#131]
this program can damage the motherboard?
I was using it on a Asus G71V with an Ultrabook profile ... it worked for a while 'and then I noticed that the CPU multiplier has halved.
In the bios POST is successful with 12x 3GHz ... in Windows goes to 6x 1.5 Ghz
I checked all the options, I have the performance profile and set the CPU to 100% and I have no other settings in Bios nor Windows, the CPU has always worked at 3GHz regularly. The Bios is locked so I can not disable anything, there are no options for EIST or SpeedStep.
I formatted and reinstalled Windows hoping it was a software error ... but now I write from just reinstalled Windows ... and the CPU is always halved. I repeat, the POST is correct.

This program can damage the motherboard? I have to throw away my pc :_( ?
Pls someone let me know...

**Comment by hirschmann** [comment]
Have you used tools like RMClock or ThrottleStop in the past?
Have you tried to reset the EC? 
(see [NBFC FAQ: Is there a way to reset my notebook if something went wrong?](https://github.com/hirschmann/nbfc/wiki/FAQ#is-there-a-way-to-reset-my-notebook-if-something-went-wrong))

**Comment by ClarkSteel** [comment]
I have never used these programs that you say or anyone else for cpu or gpu since it is a very old PC. Now that you tell me I saw that ThrottleStop is famous and after reinstalling windows for the fifth time (doing a sleepless night, I did not sleep a single minute) I tried it: the cpu back to work if I disable a thing called Prochot ... but after the pc reboot everything is as before. It is as if NBFC has changed the bios settings, which is locked. I am now completely disassembling the notebook because the battery can not be reached without disassembling everything and remaining with the motherboard in my hand ... I'm also having trouble because a very very little screw...is stripped, and now I was stuck as well. I'm afraid I have almost lost my pc.

**Comment by hirschmann** [comment]
PROCHOT is a signal which indicates the CPU is about to overheat.
This means your CPU is actually too hot, or the config you have selected set the flag accidentally. Stuff like this can happen if you use a config which was not meant to be used with your notebook model.

Resetting the EC will resolve your problems.
You don't have to disassemble your notebook to reset the EC!
Please read the FAQ again (especially the "In case the battery is not removable" part)

**Comment by ClarkSteel** [comment]
the cpu was freezing, even just because of the halfed multiplier...dropped even below 30 C°,, when it was usually around 40.
I found the EC battery and disconnected it for a few minutes ... I have reassembled the notebook and reinstalled windows (sixth time) ... now things seem back to normal, the CPU goes back to x12.
NBFC had changed somehow storing in EC on the use of the fan that did not rise more than 2200 RPM, and the CPU automatically had self-halved, for safety...I guess.
I would not have had to use this software and I will never do again.
